### DISCUSSION TITLE
Copy next/previous item with notification

### DISCUSSION THREAD

**Discussion by eggbean** [hluk/CopyQ#2389] [Q&A] [answered]
Great program, I'm so glad I finally found it. 

I have added global shortcuts for Copy next item (Ctrl-Alt-F12) and Copy previous item (Ctrl-Alt-F11) to quickly go back and forwards through the clipboard history. This would be much more useful with some sort of notification. It seems that the non-native CopyQ notification is quicker, so I would like to have that quickly come up to show the currently selected clipboard item.

How can this be scripted? Ideally it would be very quick and quick changing, coming up centre-top in a semi-translucent yellow colour.

**Comment by eggbean** [comment]
I forgot to mention, I know how to enable notifications, like https://github.com/hluk/CopyQ/discussions/1626, but I don't want to enable notifications all the time - only for these two global shortcut commands, so I suppose a custom script will be required.

**Comment by hluk** [comment, marked as answer]
Below are modified versions of the commands that show the notification ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)). You will also need to remove the old versions in the command list in Commands dialog.

```ini
[Commands]
1\Command="
    copyq:
    next()
    notification(
        '.title', 'Copied Next',
        '.message', clipboard(),
        '.id', 'copy-next-prev',
        '.time', 5000,  // hide after N milliseconds
    )"
1\GlobalShortcut=ctrl+alt+f12
1\Icon=\xf063
1\IsGlobalShortcut=true
1\Name=Copy next item
2\Command="
    copyq:
    previous()
    notification(
        '.title', 'Copied Previous',
        '.message', clipboard(),
        '.id', 'copy-next-prev',
        '.time', 5000,  // hide after N milliseconds
    )"
2\GlobalShortcut=ctrl+alt+f11
2\Icon=\xf062
2\IsGlobalShortcut=true
2\Name=Copy previous item
size=2
```
